chameleon_mailman/tasks.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, unicode_literals
+
+from celery import shared_task
+
+from django.core.mail import send_mail
+
+
+@shared_task
+def schedule_email(*args, **kwargs):
```

Comment:
Nice, that is very simple 👍 - is there any befit in returning the result?

Reply:
No, on the contrary, returning was breaking my code when testing. It seems apply_async was trying to operate on an int object if return was added. It is also possible I was using apply_async incorrectly at that time..